src/stdlib_specialfunctions_gamma.fypp

```diff
@@ -1,7 +1,6 @@
-#:set WITH_QP = False
-#:set WITH_XDP = False
 #:include "common.fypp"
-#:set CI_KINDS_TYPES = INT_KINDS_TYPES + CMPLX_KINDS_TYPES
+#:set RC_KINDS_TYPES = REAL_KINDS_TYPES[0:2]
```

Comment:
For what stands the `C` in `RC_KINDSTYPES` (I was expecting the it includes `complex`)

Reply:
Sorry, I just wanted to create a local copy of `REAL_KINDS_TYPES` with the subset, the name is indeed misleading, I'll change it.

Reply:
@jvdp1 this is what I managed:
```
#:set R_KINDS_TYPES = [KT for KT in REAL_KINDS_TYPES if KT[0] in ["sp","dp"]]
#:set C_KINDS_TYPES = [KT for KT in CMPLX_KINDS_TYPES if KT[0] in ["sp","dp"]]
#:set CI_KINDS_TYPES = INT_KINDS_TYPES + C_KINDS_TYPES
```
What do you think?